shell/platform/embedder/embedder.cc

```diff
@@ -953,6 +953,24 @@ FlutterEngineResult FlutterEngineInitialize(size_t version,
     settings.root_isolate_create_callback =
         [callback, user_data](const auto& isolate) { callback(user_data); };
   }
+  if (SAFE_ACCESS(args, log_message_callback, nullptr) != nullptr) {
+    FlutterLogMessageCallback callback =
+        SAFE_ACCESS(args, log_message_callback, nullptr);
+    settings.log_message_callback = [callback, user_data](
+                                        const std::string& tag,
+                                        const std::string& message) {
+      callback(tag.c_str(), message.c_str(), user_data);
+    };
+  } else {
```

Comment:
Earlier, the fallback on Darwin platforms syslog. With this, it will be stdout.

Reply:
Both iOS and macOS embedders include patches to rely on syslog. This won't catch third-party darwin embedders though, that's correct.

Reply:
Done.